File: stdlib/go/File.go
package main
import (
	"os"
)

func read(path dynamic) (interface{}, dynamic) {
	var content, err = os.ReadFile(path.(string));
	return content, err;
}

func write(path2 dynamic, content dynamic) {
var d1 = []byte(content.(string))
os.WriteFile(path2.(string), d1, 0644)
}

File: stdlib/js/File.js
const fs = require('fs');

class File {
    static read(file) {
        var filei = new XMLHttpRequest();
        filei.open("GET", file, false);
        filei.onreadystatechange = function ()
        {
            if(filei.readyState === 4)
            {
                if(filei.status === 200 || filei.status == 0)
                {
                    var recieved = filei.responseText;
                    return recieved;
                }
            }
        }
        filei.send(null);
    }

    static write(file, content) {
        fs.writeFile(file, content, err => {
            if (err) {
              console.error(err);
            }
        })
    }
}